test/core.test.js

```diff
@@ -110,6 +151,22 @@ lab.experiment('campaigns', async() => {
     code.expect(cookie[0].timestamp).to.exist();
   });
 
+  lab.test('setting campaign names with "/" and "|" characters', async() => {
+    server.route({
+      path: '/somecampaign',
+      method: 'get',
+      handler(request, h) {
+        return { cookie: request.getCampaigns() };
+      }
+    });
```

Comment:
Isn't this a failed test? We need to verify that we are running getCampaigns and making sure the values are right